changelog.md

```diff
@@ -2,9 +2,13 @@
 
 ## Unreleased changes
 
+### HA integration
+
+- knx_event: add `source` attribute
```

Comment:
Either use `source` and `destination` 
Or use `source_address` and `destination_address`

Reply:
It would probably break some HA automations, that's why I asked.